patches/minecraft/net/minecraft/world/entity/projectile/FishingHook.java.patch

```diff
@@ -11,12 +11,21 @@
        if (!p_37137_.m_213877_() && p_37137_.m_6084_() && (flag || flag1) && !(this.m_20280_(p_37137_) > 1024.0D)) {
           return false;
        } else {
-@@ -241,7 +_,7 @@
+@@ -241,7 +_,16 @@
  
     private void m_37171_() {
        HitResult hitresult = ProjectileUtil.m_37294_(this, this::m_5603_);
 -      this.m_6532_(hitresult);
-+      if (hitresult.m_6662_() == HitResult.Type.MISS || !net.minecraftforge.event.ForgeEventFactory.onProjectileImpact(this, hitresult)) this.m_6532_(hitresult);
++      if (hitresult.m_6662_() == HitResult.Type.MISS)
++      {
++         switch (net.minecraftforge.event.ForgeEventFactory.onProjectileImpact(this, hitresult))
++         {
```

Comment:
Formatting: In Minecraft code, the brace style should follow the decompiler output, which is braces on the same line.